### ISSUE TITLE
Invalid line "include conf.d/*.conf" in the config file when building VernemMQ from the source 

### ISSUE THREAD

**Issue by dusanb94** [vernemq/vernemq#1767]
### Environment

- VerneMQ Version: 1.11.0
- OS: Arch linux
- Erlang/OTP version (if building from source): 23
- VerneMQ configuration (vernemq.conf) or the changes from the default
- Cluster size/standalone: standalone

### Expected behavior

I expect that the compiled binary and generated configuration work properly.

### Actual behavior

I got the following error code:

```
[error] Syntax error in /vernemq/bin/../etc/vernemq.conf after line 1471 column 1, parsing incomplete
configuration error, exit
```

After inspection of the `vernemq.conf` file, I found that the following line causes the issue: 
```
include conf.d/*.conf
```
I tested build both locally on my machine with the aforementioned environment and in the Docker build using `erlang:22.2-alpine` image. I also tested with different versions of Erlang and VerneMQ, but it did not help. When I remove this line in the config file, the problem is solved. In the attachments, you can find an example of generated `vernemq.conf` (I changed the extension to `.txt` because GitHub does not support `.conf` files).
[vernemq.conf.txt](https://github.com/vernemq/vernemq/files/6254320/vernemq.conf.txt)

- Please include minimal steps to reproduce the issue, if possible.
I followed the [Quick Start](https://github.com/vernemq/vernemq#quick-start) guide from the README file.

**Comment by ioolkos** [comment]
@dusanb94 I have no idea why you have the line `include conf.d/*.conf` in `vernemq.conf`. It should not be there.
Maybe something related to Cuttlefish or Rebar3.

**Comment by drasko** [comment]
@ioolkos can you please try it, it looks like compilation is broken in the latest release. It is reproducible, several of us tried with the same error - auto-generated conf file get inserted this line which breaks the execution later (i.e. malformed conf file).

**Comment by ioolkos** [comment]
@dusanb94 @drasko the issue is minor & it's this one: https://github.com/vernemq/rebar3_cuttlefish/issues/43
I'll fix the master build later when i find time. If you need this to work right now, apply the fix given there, pinning the plugin dep to a ref, instead of master.

**Comment by dusanb94** [comment]
@ioolkos Thanks for addressing this issue. It is resolved as of #1772.
